### PR TITLE
navigation

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please review it**
navigation

### CODE REVIEW

File: Components/Navigations/navigation.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    color: #000;
+}
+.user-nav-link{
+    display: flex;
+    list-style: none;
+    flex-grow: 0;
+}
+.searchbar {
+    box-shadow: 0px 0px 1px 1px rgba(0, 0, 0, 0.2);
+    color: #858585;
```
Comment (zabihshaik) [inline Components/Navigations/navigation.css:36]: Better to use variable for this color if it is used elsewhere as well.

File: styles.css [hunk 2]
```diff
@@ -14,7 +14,7 @@ h1 {
 
 .header {
     display: flex;
-    box-shadow: 0px 1px 2px 1px rgb(245, 196, 122);
+    box-shadow: 0px 1px 2px 1px rgba(0,0,0,0.2);
     background: rgb(253, 242, 219);
```
Comment (zabihshaik) [inline styles.css:18]: Better to use variable for background here, if it's specific to this element, it's OK I guess.

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a couple of comments.

File: Components/Navigations/navigation.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+@import url("/layout.css");
+@import url("/styles.css");
+
+input {
+    box-shadow: 0px 0px 1px 1px rgba(0, 0, 0, 0.2);
+    background-position: 10px 10px;
+    background-repeat: no-repeat;
+    margin: 0 1rem 0 0px;
```
Comment (yashpatel1797) [inline Components/Navigations/navigation.css:8]: sometimes you used rem and sometimes px use only one. use px if you assign a very small value.

Comment (anshtripathi01) [reply Components/Navigations/navigation.css:8]: Ok I look into it

File: Components/Navigations/navigation.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    border: none;
+    outline: none;
+    border-left: none;
```
[the remark this answers is not in the document]
Comment (anshtripathi01) [reply Components/Navigations/navigation.css:11]: Sorry I forgot to change it

File: styles.css (hunk 2 above)
Comment (anshtripathi01) [reply styles.css:18]: Thank you for review .
I forgot to change it.

**Pull request merged**